install.sh

```diff
@@ -3048,7 +3090,7 @@ setup_new_templates_folder() {
     if ! is_dry_run; then
         if mkdir -p "${TILDE_REPLACED}/hooks"; then
             # Let this one go with or without a tilde
-            git config --global init.templateDir "$USER_TEMPLATES"
+            set_githooks_directory "$USER_TEMPLATES/hooks"
```

Comment:
Why the extra `/hooks` ?

Reply:
Please drop the extraneous `/hooks`, I believe that is what's breaking the tests.